### ISSUE TITLE
Replace RelativeLayout with ConstraintLayout

### ISSUE THREAD

**Issue by edblanco** [edblanco/TriviaGame#4]
In the dialog_prompt.xml, replace the root RelativeLayout with a constraint layout. Remove also the LinearLayout that way everything is at the same level.

**Comment by Anshu-Bijarnia** [comment]
@andrp92 are you working on this issue, or i can also give it a try? I am new to opensource and would love to contribute.

**Comment by andres-pcg** [comment]
Yes @Anshu-Bijarnia, I'm already working on it. However, we appreciate you reaching out.

¿Can we get back to you if we get more issues like this?

**Comment by Anshu-Bijarnia** [comment]
Yes @andrp92 , I would love to start with some good first issues.
